.gitignore

```diff
@@ -1,6 +1,7 @@
 build*
 xcode*
 .DS_Store
+.vscode
 src/mlpack/core/util/gitversion.hpp
 src/mlpack/core/util/arma_config.hpp
```

Comment:
I've added `.vscode` since it becomes annoying sometimes if vscode files get added by mistake. I think its fine to add this. Or should I open a separate PR for this?

Reply:
That's fine.